### DISCUSSION TITLE
【灌水】晚上乱逛看到了一个奇怪标志，新艾利都似乎没有类似的组织吧？

### DISCUSSION THREAD

**Discussion by Drs1145** [share121/inter-knot#1000] [其他]
![R-C](https://github.com/user-attachments/assets/1bf32f01-9e87-4060-aa4e-ef4b0e5588c0)
如标题，本人今天晚上在街上见到的一个挺好看的女孩身上的衣服印有这个logo，看她那样子似乎是一个兔希人（又或者是驴希人？）但是左顾右盼的跟迷路了一样，作为一位良好的新艾利都公民我自然是将她送到了治安局，本来想借此机会找她要个联系方式的结果她什么也不知道。哪怕是电话号码也没有。
我严重怀疑这是一个境外势力的渗透分子，所以特来询问绳网的各位大神是否认得这个logo。

**Comment by Erdbronie** [comment]
这是什么标志，我在新艾利都从来没见过，好可怕（bushi

**Comment by capacitor1** [comment]
哦，我貌似见过这个标志。它来自于另外一个世界...

doge

**Reply by Drs1145** [reply]
> 哦，我貌似见过这个标志。它来自于另外一个世界...

愿闻其详

**Comment by Qiling-XwX** [comment]
来自世界之外的力量（）

**Comment by ling188** [comment]
各位好心人麻烦告诉她一声，赶紧回罗德岛，另外是兔希人不是驴希人啦

**Comment by ZhuYuan-148** [comment]
治安官朱鸢已介入调…

**Comment by qiu-1214** [comment]
这是来自世界之外的力量

**Comment by Zeteor** [comment]
我靠粥吧打过来了

**Comment by SSTV-10068** [comment]
这是我司新开发的游戏。事件具体内容[请看这里](https://github.com/share121/inter-knot/discussions/1049#new_comment_form "一个链接")

**Comment by Griffin233** [comment]
这是一家医药公司，我在那边有合法职位，这样吧我把我的工号发给你，你也许可以联系我。以下是我的工号

格里芬#7526

**Comment by Drs1145** [comment]
绳友们出事了！我进空洞了！

**Comment by sekiroissekiko** [comment]
好家伙，什么世界之外的力量（
